README.md

```diff
@@ -172,6 +172,7 @@ fixing a bug or by adding a feature and open a pull request. Just make sure that
 [<img src="https://avatars.githubusercontent.com/u/70689411?s=48&amp;v=4" width="48" height="48" alt="brandeins1403">](https://github.com/brandeins1403)
 [<img src="https://avatars.githubusercontent.com/u/56362368?s=48&v=4" width="48" height="48" alt="benedicthomuth">](https://github.com/benedicthomuth)
 [<img src="https://avatars.githubusercontent.com/u/7889564?s=48&amp;v=4" width="48" height="48" alt="schwehn42">](https://github.com/schwehn42)
+[<img src="https://avatars.githubusercontent.com/u/58441634?s=400&v=4" width="48" height"48" alt="NNKKOO">](https://github.com/NNKKOO)
```

Comment:
I did not test it, but the src URL differs from the others. Is there a specific reason for that?
If not, could you change it? :) If yes: just tell me and I'll approve it 🔥

Reply:
I've tried both options and both worked for me. but I agree, having everything uniform makes sense - let me change that